README.md

```diff
@@ -975,7 +975,7 @@ easier understanding and reading of a test.
       describe '#title' do
         it 'is unique' do
           another_article = FactoryGirl.create(:article, title: article.title)
-          article.valid?
+          another_article.valid?
           expect(another_article.errors[:title].size).to eq(1)
```

Comment:
I think this actually intended to test validation on `article`, in a context when there's `another_article` around, so it's the expectation that was wrong, i.e.:
```
expect(article.errors[:title].size).to eq(1)
```
I'll fix it in a follow-up.